modules/shortcodes/quiz.php

```diff
@@ -181,7 +181,7 @@ public static function shortcode( $atts, $content = null ) {
 		}
 
 		$quiz = self::do_shortcode( $content );
-		return '<div class="quiz"' . $id . '>' . $quiz . '</div>';
+		return '<div class="jetpack-quiz"' . $id . '>' . $quiz . '</div>';
```

Comment:
For this (and all successive changes) I'd actually far prefer if instead of replacing the old class name, we append it instead.  So `quiz` becomes `jetpack-quiz quiz` -- this allows us to narrow our styling, but also doesn't break any legacy styling that existing sites (like on WPCOM) may have implemented.